kura/org.eclipse.kura.util/src/main/java/org/eclipse/kura/util/validation/PasswordStrengthValidators.java

```diff
@@ -80,6 +80,11 @@ private static Validator<String> containsBothCases(final Messages messages) {
         };
     }
 
+    public static Validator<String> requireDifferentNameAndPassword(final String identityName) {
+
+        return new PredicateValidator(v -> !v.equals(identityName), new DefaultMessages().pwdNotEqualsUsername());
```

Comment:
Would it make sense to check ignoring case?

Reply:
well... it's a choiche, Admin is not the same of AdMIn

Reply:
ok I checked, NIST 800 says that derivative password must be rejected, included same name in different case, I'll modify it.